miru/ext/nav/navigator.py

```diff
@@ -151,11 +151,16 @@ def remove_item(self, item: Item[hikari.impl.MessageActionRowBuilder]) -> Naviga
     def clear_items(self) -> NavigatorView:
         return t.cast(NavigatorView, super().clear_items())
 
-    def _get_page_payload(self, page: t.Union[str, hikari.Embed]) -> t.MutableMapping[str, t.Any]:
+    def _get_page_payload(
+        self, page: t.Union[str, hikari.Embed, t.Sequence[hikari.Embed]]
+    ) -> t.MutableMapping[str, t.Any]:
         """Get the page content that is to be sent."""
 
         content = page if isinstance(page, str) else ""
-        embeds = [page] if isinstance(page, hikari.Embed) else []
+        if isinstance(page, Sequence):
```

Comment:
Is it not possible to use `t.Sequence` here?

Reply:
You are correct, I've replaced it now!